hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestCreateTable.scala

```diff
@@ -641,4 +641,26 @@ class TestCreateTable extends HoodieSparkSqlTestBase {
          |""".stripMargin
     )
   }
+
+  if (SPARK_VERSION.startsWith("3")) {
```

Comment:
should be HoodieSparkUtils.gteqSpark3_2 ? since HoodieCatalog would only be used after 3.2.0

Reply:
Thanks, that's much better.